src/Foundatio/Storage/IFileStorage.cs

```diff
@@ -18,7 +19,61 @@ public interface IFileStorage : IHaveSerializer, IDisposable {
         Task<bool> CopyFileAsync(string path, string targetPath, CancellationToken cancellationToken = default);
         Task<bool> DeleteFileAsync(string path, CancellationToken cancellationToken = default);
         Task DeleteFilesAsync(string searchPattern = null, CancellationToken cancellation = default);
```

Comment:
Seems like you'd want a result here or a long count on how many were removed

Reply:
Guess we could do that since we are already breaking things.